aries-site/src/components/cards/LinkedCard.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Link from 'next/link';
+
+import { nameToPath } from '../../utils';
+import { ContentCard } from '.';
+
+export const LinkedCard = ({ topic }) => {
```

Comment:
If CardGrid is enforcing the usage of a linked card, do we need this component? Maybe we could just do the wrapping of a ContentCard within CardGrid directly.

Reply:
Good call. Will re-evaluate.

Reply:
Implemented.